src/index.html

```diff
@@ -21,16 +17,19 @@
       integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
       crossorigin="anonymous"
     />
-    <link rel="manifest" href="/manifest.json" />
+    <link
+      rel="manifest"
+      href="/manifest.json" />
     <link
       rel="shortcut icon"
-      href="<%= htmlWebpackPlugin.options.publicFolder %>favicon.png"
+      href="/favicon.png"
     />
     <title>Stormkit - Console</title>
   </head>
   <body>
     <noscript> You need to enable JavaScript to run this app. </noscript>
     <div id="root"></div>
     <div id="modal-root" role="dialog" aria-modal="true"></div>
+    <script type="module" src="./index.tsx"></script>
```

Comment:
**Note** This is slightly different than webpack. Vite treats the `index.html` file as the entry point and then replaces these imports during build time. Therefore this line is required to launch the application.